bodhi/server/scripts/untag_branched.py

```diff
@@ -14,7 +14,7 @@
 """
 Used to remove the pending and testing tags from updates in a branched release.
 
-Since a seperate task mashes the branched stable repos, this will leave
+Since a separate task mashes the branched stable repos, this will leave
```

Comment:
I recommend fixing this spelling error in a separate pull request. This way we can keep the concerns separate and make this PR focus only on making the batched stuff work. Also, the spelling error PR can help pad your GitHub stats ☺